packages/at_onboarding_cli/lib/src/cli/auth_cli.dart

```diff
@@ -680,9 +717,119 @@ Future<void> approve(ArgResults ar, AtClient atClient) async {
     final response = await atAuthBase
         .atEnrollment(atClient.getCurrentAtSign()!)
         .approve(decision, atLookup);
-    // 'enroll:approve:{"enrollmentId":"$enrollmentId"}'
+
     stdout.writeln('Server response: $response');
+
+    approved++;
+
+    if (limit != null && approved >= limit) {
+      return approved;
+    }
+  }
+  return approved;
+}
+
+Future<int> autoApprove(ArgResults ar, AtClient atClient) async {
```

Comment:
New feature.
- if requested, calls `approve` to approve any existing matching enrollment requests, and respecting the approval count limit while doing so
- listen for new enrollment request notifications
- when one is received, and it matches the supplied regex filter(s), then
  - approve it
  - check the approved count vs limit
  - if limit has been reached, complete the completer so the `await completer.future` completes
  - clean up the subscription